src/styles/main.scss

```
@@ -0,0 +1,63 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.clock {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+
+  &__item {
+    position: relative;
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
```

Роби відступи між позиціонуванням і стилем блоку
```suggestion
    width: 30px;
    
    border-radius: 50%;
```